### DISCUSSION TITLE
Unable to open PDF link from the mobile app

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#466] [Feature request]
I've a course in which there is a PDF and that PDF is having some links. 

**On Portal:** If I try to open the links from the PDF, it gets open.
**On mobile App:** If I try to open the links from the PDF, nothing happens.

**_I want to know how the links in PDF can be opened from the mobile app?_** 
This a requirement by client so need to know why it is not working and what should I've to do to make it in working state?

Mobile App version and portal version are 4.6.

cc: @swayangjit

**Comment by swayangjit** [comment]
Thanks  @nidhikachhawa24  for raising this query. This is the current capability. If you are interested to make these changes and contribute we can help you.

**Reply by nidhikachhawa24** [reply]
@swayangjit 
I would love to contribute this change and would need some guidance on how to go about it ?

**Reply by swayangjit** [reply]
@nidhikachhawa24 You can start debugging the issue starting with if you are getting any errors while clicking on the link . If No then you have to debug  the [pdf player](https://github.com/Sunbird-Knowlg/sunbird-pdf-player)

**Reply by Maheshk-Trigyn** [reply]
@nidhikachhawa24 / @swayangjit - The pdf loads fresh in viewer, i.e., first time.  In case we have any navigation hyperlinks in pdf that won't open. This is presently the issue we have.  Did any of you are aware of this?   The Pdf player doesn't have any fix, on this links issue for long, I believe.  Any plans from sunbird to fix this issue in future?
